src/firmware_new/tools/communication/robot_communicator.h

```diff
@@ -1,96 +0,0 @@
-#pragma once
```

Comment:
has this file been deleted?

Reply:
yeah its intentional, I didn't want to update it to use the ProtoMulticast*** because it would immediately change. It will be properly replaced within the next week as part of https://github.com/UBC-Thunderbots/Software/issues/1367